datadog-service/0.16.0/README.md

```diff
@@ -0,0 +1,118 @@
+# datadog-service
```

Comment:
All the README.md files in this PR are pulled from the https://github.com/keptn-sandbox/datadog-service [according to the tag](https://github.com/keptn-sandbox/datadog-service/tags) with some changes.